### PR TITLE
Features/events

### PR DESCRIPTION
**Pull request by fcurella**
Work in Progress. Do not merge yet.

### CODE REVIEW

**Comment by fcurella** [comment]
@jhogue:  I'd need HTML/CSS for the following templates:
- event_detail.html
- eventcategory_list.html (this will contain a paginated list of all categories for events)
- eventlocation_list.html (this will contain a paginated list of all locations created for events)

**Comment by jhogue** [comment]
Was able to do some work with this today. Made two commits, one for the Event templates themselves and another for related changes to other non-Event templates. Minor changes to maintain consistency between single page templates. 

Added "waitforit.html" as our "Coming Soon" placeholder template in the Initial review.

**Comment by jhogue** [comment]
Was going to work on the categories template, but when I tried to add one through the admin, I got an error that the template does not exist: "FieldDoesNotExist at /admin/events/eventcategory/ EventCategory has no field named 'created'". I was able to create a sample location. 

Also, I dont know how much we want to worry about this yet, but the admin lists many objects by a generic name, like "EventLocation Object" in lists and drop downs. If I had more than one sample, it'd be tough to pick the right one.

File: static/sass/_layout.scss [hunk 1]
```diff
@@ -344,7 +353,6 @@
      */
     $names: python, psf, docs, pypl, jobs, shop;
     $colors: $blue, $psf, $yellow, $green, $purple, $red;
-    $type-colors: $grey-lightest, $grey-lightest, $grey-dark, $grey-lightest, $grey-lightest, $grey-lightest;
```
Comment (jhogue) [inline static/sass/_layout.scss]: Keep this line in.

File: static/sass/_layout.scss [hunk 2]
```diff
@@ -359,7 +367,7 @@
                 @include pe-border( $color-top: lighten($this-color, 10%), $color-left: lighten($this-color, 10%), $color-bottom: darken($this-color, 20%), $color-right: darken($this-color, 10%) );
                 
                 & > a {
-                    color: nth($type-colors, $i);
```
Comment (jhogue) [inline static/sass/_layout.scss]: Keep this line in as well.

File: static/sass/_layout.scss [hunk 3]
```diff
@@ -359,7 +367,7 @@
...
+                    color: $grey-lightest;
```
Comment (jhogue) [inline static/sass/_layout.scss]: Remove this one.

**Comment by fcurella** [comment]
I'm going to squash commit this one before merging. whoever is tracking this branch will have to delete it and re-fetch it

**Comment by jhogue** [comment]
Ok, I am... Let me know. I have a little more to add if you can wait.

On Fri, Mar 8, 2013 at 3:53 PM, Flavio Curella notifications@github.comwrote:

> I'm going to squash commit this one before merging. whoever is tracking
> this branch will have to delete it and re-fetch it
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/proevo/pythondotorg/pull/7#issuecomment-14644850
> .

## 

_J. Hogue_: Design Director at Project Evolution
t: 888.456.0467 | c: 401.831.1963

**Comment by fcurella** [comment]
commits are squashed. all clear.

**Pull request merged**